wvd-sh/WVD scaling script/basicScale-MSI.ps1

```diff
@@ -497,27 +498,40 @@ if ($hostpoolInfo.LoadBalancerType -eq "DepthFirst") {
                                 }
                             }
                         }
+
+
                         $VMName = $sessionHost.Split(".")[0]
+                          #Check the Session host is in maintainance
+                                $VmInfo = Get-AzureRmVM | Where-Object { $_.Name -eq $VMName }
+                                if($VmInfo.Tags.Keys -contains $maintainanceTagName){
+                                Write-Log 1 "Session Host is in Maintainace: $($sessionHost | out-string)"
```

Comment:
Replace "Maintainace" with "maintenance"

Reply:
We have replaced in both scripts